samples/unit-tests/series/update/demo.html

```diff
@@ -1,4 +1,4 @@
-<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/highchart.js"></script>
```

Comment:
Wrong file name => test fails in utils.

Reply:
Test does not fails for me. I also do not find any use of Highstock in the demo.js.